static/js/composebox_typeahead.js

```diff
@@ -1094,19 +1094,19 @@ export function initialize() {
 
```

Comment:
The approach I took here is one of possible approaches, I am not sure if this is the beat approach to do it. I can checking for better options.

`New` will always be visible to the user if there are no other options, even if `New` does not match the query entered by the user.